assets/css/carousel.css

```diff
@@ -0,0 +1,119 @@
+/*
```

Comment:
Please fix trailing whitespaces in this file!

Reply:
Done 🙂